crates/assists/src/utils.rs

```diff
@@ -21,8 +21,7 @@ use crate::{
     ast_transform::{self, AstTransform, QualifyPaths, SubstituteTypeParams},
 };
 
-pub use insert_use::MergeBehaviour;
-pub(crate) use insert_use::{insert_use, ImportScope};
+pub use insert_use::{insert_use, ImportScope, MergeBehaviour};
```

Comment:
JFYI, we've touched the question of moving those utils from `assists` into a separate shared crate already in https://github.com/rust-analyzer/rust-analyzer/pull/6544#issue-520677765
but did not take any actions on that yet.

Might consider this here, if needed.

Reply:
Not strictly needed, but will be good, yeah (probably in separate PR)